### DISCUSSION TITLE
Diksha offline Desktop app for raspberry pi 4 Ubuntu OS 22.04 LTS

### DISCUSSION THREAD

**Discussion by Harish192** [Sunbird-Ed/Community#35] [Enhancement request]
As part of the experience center, We are planning to set up the offline Desktop app for the Raspberry Pi 4 device with the Ubuntu 22.04 operating system, but the  [current debian ](https://ntpproductionall.blob.core.windows.net/ntp-production-offlineinstaller/latest/DIKSHA_4.9.0_linux64bit.deb) build is not compatible for this device
Kindly request you to have a look onto this.

Raspberry pi 4 Details:
Processor : 64 bit quad core cortex- A72 ARM
RAM: 4 GB
OS : Ubuntu 22.04 LTS

Error message:

1. $ sudo apt install /home/raspberrypi/Downloads/DIKSHA_4.9.0_linux64bit.deb
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'diksha:amd64' instead of '/home/raspberrypi/Downloads/DIKSHA_4.9.0_linux64bit.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:
The following packages have unmet dependencies:
 diksha:amd64 : Depends: libgtk-3-0:amd64 but it is not installable
                Depends: libnotify4:amd64 but it is not installable
                Depends: libnss3:amd64 but it is not installable
                Depends: libxss1:amd64 but it is not installable
                Depends: libxtst6:amd64 but it is not installable
                Depends: libatspi2.0-0:amd64 but it is not installable
                Depends: libuuid1:amd64 but it is not installable
                Depends: libsecret-1-0:amd64 but it is not installable
                Recommends: libappindicator3-1:amd64 but it is not installable
E: Unable to correct problems, you have held broken packages.

2. sudo dpkg -i /home/raspberrypi/Downloads/DIKSHA_4.9.0_linux64bit.deb
[sudo] password for raspberrypi:
dpkg: error processing archive /home/raspberrypi/Downloads/DIKSHA_4.9.0_linux64bit.deb (--install):
 package architecture (amd64) does not match system (arm64)
Errors were encountered while processing:
 /home/raspberrypi/Downloads/DIKSHA_4.9.0_linux64bit.deb

3. $ sudo aptitude install /home/raspberrypi/Downloads/DIKSHA_4.9.0_linux64bit.deb
Couldn't find any package whose name or description matched "/home/raspberrypi/Downloads"
Unable to apply some actions, aborting

**Comment by sharathkashyap** [comment]
@Harish192 The Enhancement request will be scheduled for 4.10.2 considering its priority. 
cc: @luckynvdu
